api/src/opentrons/protocol_api/labware.py

```diff
@@ -773,11 +773,14 @@ def set_offset(self, x: float, y: float, z: float) -> None:
                 you must either use ``set_offset()`` on all of them or none of them.
             * - 2.14–2.17
               - ``set_offset()`` is not available, and the API raises an error.
-            * - 2.18 and newer
+            * - 2.18-2.22
```

Comment:
```suggestion
            * - 2.18–2.22
```
Doesn't match the above. (En dashes are annoying in monospace, I know. Sphinx can also convert `--` to an en dash.)